app/services/data_migrations/set_missing_provider_relationship_permissions.rb

```diff
@@ -0,0 +1,26 @@
+module DataMigrations
+  class SetMissingProviderRelationshipPermissions
+    TIMESTAMP = 20210607134802
+    MANUAL_RUN = true
+
+    def change
+      Course.all.each do |course|
+        next if ratified_by_provider?(course) || provider_relationship_permissions_exist?(course)
+
+        ProviderRelationshipPermissions.new(training_provider: course.provider,
```

Comment:
do we want to also add `setup_at` to this?

Reply:
I think we do want users to explicitly set up these permissions, as alternatively we might end up having users testing in sandbox unexpectetly start seeing applications that they were not expecting show up.

Reply:
double checked sandbox and its only 2 providers so its ok